src/main/java/com/softserve/academy/spaced/repetition/security/service/AuthenticationRestService.java

```diff
@@ -58,9 +58,9 @@
 
     public HttpHeaders getAuthHeaders(String email, String password, String captcha, Device device) throws UserStatusException {
         ReCaptchaResponseDto reCaptchaResponseDto = reCaptchaApiService.verify(captcha);
-        if (!reCaptchaResponseDto.isSuccess()) {
-            throw new BadCredentialsException("reCaptcha");
-        }
+//        if (!reCaptchaResponseDto.isSuccess()) {
```

Comment:
remove this comments

Reply:
no ^_^

Reply:
So, if it is necessary, why did you put comments?